addOns/grpc/src/test/java/org/zaproxy/addon/grpc/internal/ProtoBufMessageEncoderUnitTest.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2024 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.addon.grpc.internal;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.nullValue;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ProtoBufMessageEncoderUnitTest {
+
+    private ProtoBufMessageEncoder encoder;
+
+    @BeforeEach
+    void setUp() {
+        encoder = new ProtoBufMessageEncoder();
+    }
+
+    @Test
+    void testStartEncodingWithEmptyInput() throws Exception {
```

Comment:
Better name as `should...` like all other tests (or 99% of them).

Reply:
Should start with lower case though.

Reply:
ohh ok